frontend/src/services/api.ts

```diff
@@ -227,3 +226,8 @@ export const resetPassword = async (
     throw error;
   }
 };
+
+export const fetchEventFiltered = async (params: any) => {
+  const response = await axiosInstance.get(`/api/events/filter/?${params}`);
```

Comment:
backend need adjustment as unrelated data is being return, search for Toronto, on return both toronto and columbus and same vice versa for columbus return both toronto and columbus instead of just 1 event return.

The price input has mismatch in width 
![Screenshot 2024-05-30 at 6 28 17 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/68d76597-1f9e-4531-9424-38b906414025)
![Screenshot 2024-05-30 at 6 28 33 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/f53c2e41-4c61-4954-a92f-4d21d4cc2c40)
![Screenshot 2024-05-30 at 6 29 36 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/118f829a-d596-42d2-bfb1-566cc9fefa90)

Reply:
Search bar connected to Discover page
https://www.loom.com/share/9c410734ed61460fa910982f047fa689?sid=647e8101-0264-4bd4-afd3-39543fbc4942